pandas/tests/io/json/test_ujson.py

```diff
@@ -1,7 +1,7 @@
 try:
     import json
 except ImportError:
-    import simplejson as json
+    import simplejson as json  # type: ignore
```

Comment:
this is a known mypy error, so `# type: ignore` is fine here, but can you add a comment. i.e.

```python
# https://github.com/python/mypy/issues/1153
# error: Name 'json' already defined (by an import)
```

Reply:
sure, will add those comments